src/services/EvalFactor.jl

```diff
@@ -411,7 +412,8 @@ function evalPotentialSpecific( Xi::AbstractVector{<:DFGVariable},
 
   # view on elements marked for nullhypo
   addEntrNH = view(addEntr, nhmask)
-  spreadDist = spreadNH*sqrt(calcCovarianceBasic(mani, addEntr))
+  # spreadDist = spreadNH*sqrt(calcCovarianceBasic(mani, addEntr))
+  spreadDist = spreadNH*calcStdBasicSpread(getVariableType(Xi[sfidx]), addEntr)
```

Comment:
thanks for fixing this and other use in IIF.

Reply:
Maybe a better verb-noun is `calcSpreadBasic`